tests/constants.js

```diff
@@ -9,10 +9,13 @@ module.exports = {
   API_HOST: 'localhost',
 
   // connection information for the couchdb instance
-  COUCH_PORT: 5984,
+  // locally we spin up a different CouchDB for e2e tests
+  COUCH_PORT: IS_TRAVIS ? 5984 : 4984,
   COUCH_HOST: 'localhost',
 
   // test database to avoid writing to the dev db
+  // TODO: we don't need to do this anymore since it's in its own docker container
+  // HOWEVER, we have dodgy hard-coded work in login.js around medic-test
   DB_NAME: 'medic-test',
```

Comment:
We should change this back. However, my preferred way of doing this is to fix redirect support and there is quite a bit of work in the way so we'll leave it like this for now